include/hermes/VM/JSLib/RuntimeCommonStorage.h

```diff
@@ -38,7 +38,7 @@ struct RuntimeCommonStorage {
   MockedEnvironment tracedEnv;
 
   /// PRNG used by Math.random()
-  std::minstd_rand randomEngine_;
+  std::mt19937_64 randomEngine_;
```

Comment:
Have you considered using the 64 bit LCG like the one in the article you shared in your PR description?
```
typedef std::linear_congruential_engine<
    uint64_t, 6364136223846793005U, 1442695040888963407U, 0U>
        knuth_lcg;  /* Knuth's preferred 64-bit LCG */
```
I have slight preference for this one since it is easier to reason about than the Mersenne twister and also, minstd_rand is just the 32 bit version of it and it will therefore be less of a change to just use a 64 bit LCG.

Reply:
I ran benchmarks with `knuth_lcg` (updated in PR description above) and it appears to appear slower for some reason.